src/main/java/com/cloudbees/jenkins/support/impl/BuildQueue.java

```diff
@@ -101,6 +101,7 @@ public void writeTo(OutputStream os) throws IOException {
               out.println("----");
               out.println();
             }
+            out.println("Is quieting down: " + Helper.getActiveInstance().isQuietingDown());
```

Comment:
🐝 
Suggestion:
out.println("Is quieting down: " + Helper.getActiveInstance().isQuietingDown() + "new builds are not being executed");